plugins/acss-kit/assets/foundation/sass/_globals.scss

```
@@ -0,0 +1,166 @@
+
+sup {
+  vertical-align: super;
+  font-size: smaller;
+}
+
+blockquote {
+  --blockquote-padding: 1rem;
+  --blockquote-border-width: 0.3125rem;
+  --blockquote-border-style: solid;
+  --blockquote-border-color: var(--color-border);
+  padding: var(--blockquote-padding);
```

Stylelint (17.10.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)